compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/scene/skia/WindowSkiaLayerComponent.desktop.kt

```diff
@@ -47,6 +50,13 @@ internal class WindowSkiaLayerComponent(
             // apply `checkNotNull` for "non-null" field.
             checkNotNull(mediator.accessible)
         },
+        properties = run {
+            val defaultProperties = SkiaLayerProperties()
+
+            SkiaLayerProperties(
+                isVsyncEnabled = renderingSettings.isVsyncEnabled ?: defaultProperties.isVsyncEnabled,
+            )
+        },
```

Comment:
```suggestion
        properties = SkiaLayerProperties(
            isVsyncEnabled = renderingSettings.isVsyncEnabled ?: SkiaLayerProperties.Default.isVsyncEnabled,
        ),
```

Reply:
There can be multiple properties in future, this approach will reconstruct SkiaLayerProperties every single time. There is no `Default` in `SkiaLayerProperties.Companion`